scripts/linters/general_purpose_linter_test.py

```diff
@@ -657,60 +556,6 @@ def test_invalid_use_of_object(self):
         self.assertEqual('Bad pattern', lint_task_report.name)
         self.assertTrue(lint_task_report.failed)
 
-    def test_invalid_use_of_range(self):
-        linter = general_purpose_linter.GeneralPurposeLinter(
-            [INVALID_RANGE_FILEPATH], FILE_CACHE)
```

Comment:
Remove `INVALID_RANGE_FILEPATH` file if it's no more required, ditto for other cases.

Reply:
Cleaned these variables up as well as the test files associated with them. Done